src/webots/engine/WbSimulationWorld.hpp

```diff
@@ -19,9 +19,11 @@
 // Description: world with physics/kinematic simulation
 //
 
+#include <QtCore/QElapsedTimer>
 #include <QtCore/QList>
 #include <QtCore/QMutex>
-#include <QtCore/QTime>
+#include <QtCore/QTimer>
```

Comment:
Are you sure this is needed?
```suggestion
```

Reply:
Yes, we are still using a QTimer in this class (but  maybe a forward declaration is sufficient, I will check).